attributes/network/11_proof.adoc

```diff
@@ -0,0 +1,32 @@
+= Proof Attribute (#11)
```

Comment:
This attribute number is already taken in #73, and @hansl has taken #73 in #86.

You should probably use `14`!